Fix back button not recorded on tizen (#843)
bitmovin/bitmovin-player-ui: CHANGELOG.md

The format is based on [Keep a Changelog](http://keepachangelog.com/)
and this project adheres to [Semantic Versioning](http://semver.org/).

## [Unreleased]

### Fixed

> - In TV spatial navigation, the BACK button could be swallowed by the UI and not reach the player when the controls were already hidden

Reviewer: > ... and not reach the player ...

Why the player? Isn't that rather an application concern?

```suggestion
- In TV spatial navigation, the BACK button could be swallowed by the UI and not reach the application when the controls are already hidden
```

Author: Good catch! Address in c7b231a7e2c631778b0fb62416f811c031887642
